src/EditorFeatures/CSharpTest/NavigateTo/NavigateToSearcherTests.cs

```diff
@@ -86,12 +122,11 @@ public async Task NotFullyLoadedOnlyMakesOneSearchProjectCallIfValueReturned()
                 AsynchronousOperationListenerProvider.NullListener,
                 callbackMock.Object,
                 pattern,
-                searchCurrentDocument: false,
                 kinds: ImmutableHashSet<string>.Empty,
                 CancellationToken.None,
                 hostMock.Object);
 
-            await searcher.SearchAsync(CancellationToken.None);
+            await searcher.SearchAsync(searchCurrentDocument: false, CancellationToken.None);
```

Comment:
it was much cleaner impl-wise to not have 'searchCurrentDocument' be state in the searcher, but rather part of the request that flowed in.